clang/test/SemaCXX/warn-unused-filescoped.cpp

```diff
@@ -76,10 +76,33 @@ struct S {
   struct SVS : public VS {
     void vm() { }
   };
+
+  struct CS {
+    CS() {}
+    CS(bool a) {}
+    CS(int b) {} // expected-warning{{unused constructor 'CS'}}
+    CS(float c);
+  };
+
+  struct DCS : public CS {
+    DCS() = default; // expected-warning{{unused constructor 'DCS'}}
+    DCS(bool a) : CS(a) {} // expected-warning{{unused constructor 'DCS'}}
```

Comment:
Is the name of the constructor actually useful in this diagostic? It will always be the name of the surrounding class, won't it?

Reply:
Yes, you are right, it will always be the same. I left it for consistency with the previous message but I agree that it doesn't provide useful information. Would you prefer the message to be "unused constructor"?